client/modules/IDE/components/ConsoleInput.jsx

```diff
@@ -24,75 +24,116 @@ function ConsoleInput({ theme, dispatchConsoleEvent, fontSize }) {
       inputStyle: 'contenteditable'
     });
 
-    cmInstance.current.on('keydown', (cm, e) => {
+    cmInstance.current.getWrapperElement().style['font-size'] = `${fontSize}px`;
+
+    return () => {
+      if (cmInstance.current) {
+        cmInstance.current = null;
+      }
+    };
```

Comment:
Probably don't need a cleanup here if all that it's doing is clearing a local variable.  If there's a destroy or cleanup function on the CM instance we would want to call that though.